lib/matplotlib/axis.py

```diff
@@ -1552,9 +1559,11 @@ def set_label_text(self, label, fontdict=None, **kwargs):
 
     def set_major_formatter(self, formatter):
         """
-        Set the formatter of the major ticker
+        Set the formatter of the major ticker.
 
-        ACCEPTS: A :class:`~matplotlib.ticker.Formatter` instance
+        Parameters
+        ----------
+        formatter : ~matplotlib.ticker.Formatter
```

Comment:
Sorry for my ignorance and laziness to not check the render  - does this resolve to a link?

Reply:
Apparently it doesn't (https://3226-7439715-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/_as_gen/matplotlib.axis.Axis.set_major_formatter.html#matplotlib-axis-axis-set-major-formatter), but some minimal testing (on another project) shows that napoleon (instead of numpydoc) does resolve it.  Which, I guess, is a good reason to revive https://github.com/matplotlib/matplotlib/pull/5743 (unless I missed some way to achieve this with numpydoc).

Reply:
I'll ask one of the numpydoc devs. In the meantime, let's move forward on that without this sorted out. This is a great improvement on what we had before.